impala/hiveserver2.py

```diff
@@ -964,10 +972,10 @@ def _operation(self, kind, request):
         return self._get_operation(resp.operationHandle)
 
     def _log_request(self, kind, request):
-        log.debug('%s: req=%s', kind, request)
+        log.info('%s: req=%s', kind, request)
```

Comment:
Is this leftover debugging code? This is going to generate a lot of noise right?

Reply:
Yes, reverted it.